coalib/bearlib/aspects/__init__.py

```diff
@@ -54,6 +54,25 @@ def __init__(self, module):
                 subaspect = getattr(submod, submodname)
                 setattr(self, submodname, subaspect)
 
+    def get(self, aspectname, multi_find=False):
+        """
+        Wrapper for ``__getitem__`` that doesn't raise exception.
```

Comment:
I am not sure whether to make `multi_find` enabled by default because it will make the result type not consistent (either aspectclass or **list** of aspectclass)